consensus/safety-rules/src/safety_rules_manager.rs

```diff
@@ -75,19 +75,32 @@ pub fn storage(config: &SafetyRulesConfig) -> PersistentSafetyStorage {
             panic!(
                 "Safety rules storage is not initialized, provide an initial safety rules config"
             )
+        };
+
+        // Ensuring all the overriding consensus keys are in the storage.
+        let timer = Instant::now();
+        for blob in config
```

Comment:
I don't think this path is executed on every restart, it's only executed when the storage is empty? i.e. line 49 returns err

Reply:
i believe it is? L49 and L80 are at the same indentation level, and there's no `return` in block L49-78.

Reply:
this block of line 55-102 is an else branch of line 50? https://github.com/aptos-labs/aptos-core/blob/869faed30c3868aec7879fc30701d545b841d745/consensus/safety-rules/src/safety_rules_manager.rs#L50

Reply:
nvm I'm noob